Madalina_Zbereanu/Week2/Demo/Program.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Domain.Domain;
+using Domain.Domain.CreateRestauratOp;
+using Domain.Models;
+using Infrastructure.Free;
+using LanguageExt;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using static Domain.Domain.CreateRestauratOp.CreateRestaurantResult;
+
+namespace Demo
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var serviceCollection = new ServiceCollection();
+            serviceCollection.AddOperations(typeof(Restaurant).Assembly);
+            var serviceProvider = serviceCollection.BuildServiceProvider();
+
+            var newRestaurant =
+                from restaurantResult in RestaurantDomain.CreateRestaurant("mcdonalds")
+                let restaurant = (restaurantResult as RestaurantCreated)?.Restaurant
+                from newMenu in RestaurantDomain.CreateMenu(restaurant, "burgers", MenuType.Meat)
```

Comment:
Can you also try to expand this expression with the CreateIngredient and CreateMenuItem? 

As you play with it, can you also try to improve their implementations?